java/ruleXtract/src/it/resources/CF.rulextract.patterns

```diff
@@ -0,0 +1,15 @@
+# source pattern constraints
+# nocc is the minimum number of occurrences of a hierarchical
+# rule in the training data
+# ntrans is the maximum number of translations per source
```

Comment:
nocc and ntrans are not explicitly used in this config, so these comments are somewhat odd. You could put commented line showing usage examples